backend/build.gradle.kts

```diff
@@ -35,6 +36,7 @@ dependencies {
     implementation(project(":dto_common"))
 
     testImplementation("org.testcontainers:mongodb:1.16.2")
+    testCompileOnly("io.micronaut.reactor:micronaut-reactor")
```

Comment:
Is this needed if it is already a `compileOnly` dependency?

Reply:
jap, without it throws an error that the method/class isn't found